sunpy/coordinates/tests/test_frames.py

```diff
@@ -509,3 +510,40 @@ def test_angular_radius_no_obstime():
     coord = Helioprojective(0*u.deg, 0*u.deg, 5*u.km, obstime=None, observer="earth")
     with pytest.raises(ValueError, match=r"The observer must be fully defined by specifying `obstime`."):
         coord.angular_radius
+
+
+@pytest.fixture
+def off_limb_coord():
+    frame = Helioprojective(observer='earth', obstime='2020-01-01')
+    return SkyCoord(Tx=[-1000, 300, 1000]*u.arcsec, Ty=[-1000, 300, 1000]*u.arcsec, frame=frame)
+
+
+@pytest.mark.parametrize('screen_class', [
+    SphericalScreen,
+    PlanarScreen,
+])
+def test_screen_classes(off_limb_coord, screen_class):
```

Comment:
Can this include a test of `only_off_disk=True`?

Reply:
I think I added one.